src/pages/new-chat/new-chat.html

```diff
@@ -10,9 +10,15 @@
 
 <ion-content class="new-chat">
   <ion-list class="users">
-    <button ion-item *ngFor="let user of users | async" class="user" (click)="addChat(user)">
+    <button ion-item *ngFor="let user of users | async | search:searchPattern:'profile.name'" class="user" (click)="addChat(user)">
```

Comment:
No. 
Your current code is as follow:
```ts
let users = Observable.of([ ... ]); // Filled from Mongo.ObservableCollection

users.subscribe((usersArr) => { // this is the async pipe
    usersArr.filter(...); // This is your filter
});
```

But you can actually do perform the same, but with RxJS Operator:
```ts
let users = Observable.of([ ... ]); // Filled from Mongo.ObservableCollection

users.filter(filterFn /* this is the filter */).subscribe((usersArr) => { 
    // usersArr is now filtered 
});
```